### PR TITLE
* #5: Added user guide.

### LINKED ISSUE #5: Add documentation
We need to add a project description to the README file and also a short user guide.

### PR DESCRIPTION
**Pull request by AnastasiiaSergienko**
Closes #5

### CODE REVIEW

File: doc/user_guide/user_guide.md [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+  ```shell
+  java -jar virtual-schema-installer-0.1.0.jar --dialect mysql --jdbc-driver-name mysql-connector.jar --jdbc-driver-path target/mysql-driver --exa-host localhost --exa-port 32769 --exa-bucketfs-port 32770 --exa-schema-name MY_SCHEMA --exa-adapter-name MY_ADAPTER_SCRIPT --exa-connection-name MY_JDBC_CONNECTION --exa-virtual-schema-name MYSQL_VIRTUAL_SCHEMA_1 --source-host 172.17.0.1 --source-port 32783 --property CATALOG_NAME='MYSQL_SCHEMA' --property TABLE_FILTER='SIMPLE_TABLE' --credentials-file /tmp/installer_credentials
+  ```
+  
+## Things to Be Aware of
+
+* Currently, all files are uploaded to the `/buckets/bfsdefault/drivers/jdbc` bucket directory.
+* The installer downloads a virtual Schema Adapter JAR from Exasol GitHub and saves it in the default temporary directory.
+* The installer only creates a new Exasol schema if a schema with a specified named doesn't exist.
+* The installer creates Exasol schema, adapter, connection and virtual schema names with quotation. It means the names you specify on the console are case-sensitive.
```
Comment (redcatbear) [inline doc/user_guide/user_guide.md:85]:
```suggestion
* The installer uses quotes when creating Exasol schema, adapter, connection and Virtual Schema name. That means the names you specify on the console are case-sensitive.
```

**Review by redcatbear — CHANGES_REQUESTED** [review]
Reviewed with findings.

Using aliases makes the commands more readable and also scripts more portable. Also, in cross references always use anchors in Markdown please.

File: doc/user_guide/user_guide.md [hunk 2]
```diff
@@ -0,0 +1,87 @@
+# Virtual Schema Installer User Guide
```
[the remark this answers is not in the document]
Comment (AnastasiiaSergienko) [reply doc/user_guide/user_guide.md:1]: Done

[the remark this answers is not in the document]
Comment (redcatbear) [reply doc/user_guide/user_guide.md:1]: Did you push this change? I don't see it.

**Review by redcatbear — CHANGES_REQUESTED** [review]
One change seems to be missing: introductory paragraph in user guide.

**Review by redcatbear — APPROVED** [review]

**Pull request merged**